src/pages/components/button.tsx

```diff
@@ -42,7 +42,7 @@ const disabledComponents = [
   { id: 'tertiary', comp: Button.tertiary },
   {
     id: 'negative',
-    comp: Button.negative,
+    comp: Button.inverted,
```

Comment:
The content itself still has a heading and example of a "negative button". This should probably be changed when updating the component itself lest we forget.

Reply:
Let's review the site after this MR and do necessary updates immediately after.